.merge_file_a02944

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from "react";
+import styled from "@emotion/styled";
+import { RouteComponentProps } from "@reach/router";
+import test from "../static/svg/icon-power-off.svg";
+
+const GilbertColor = styled.div`
+  font-family: "Gilbert Color", serif;
+  font-size: 45px;
+  font-weight: bold;
+  font-style: normal;
+`;
+
+const Gilbert = styled.div`
+  font-family: "Gilbert", serif;
+  font-size: 45px;
+  font-weight: bold;
+  font-style: normal;
+`;
+
+export const Example: FC<RouteComponentProps> = () => (
+  <div>
+    <GilbertColor>
+      CHAT ponúka individuálne arteterapie pre deti, dospelých a rodiny.
+    </GilbertColor>
+    <Gilbert>
+      CHAT ponúka individuálne arteterapie pre deti, dospelých a rodiny.
+    </Gilbert>
+
+    <img src={test} alt="svg" />
+  </div>
+);
```

Comment:
I think filename have strange name ".merge_file_a02944" and I don't think that file should be here